src/bpf/bpfd-manager.c

```diff
@@ -0,0 +1,448 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <sys/wait.h>
+
+#include "bpf-dlopen.h"
+#include "fileio.h"
+#include "sd-daemon.h"
+#include "sd-messages.h"
```

Comment:
please put `sd-*.h` headers first.

(they are out public API, and hence sit somewhere between the public includes we do via `#include <…>` and our local includes we do via `#include "…"`)